### DISCUSSION TITLE
jemalloc

### DISCUSSION THREAD

**Discussion by Z2Flow** [postmodern/ruby-install#444] [Q&A]
On macOS, how do I install ruby with jemalloc?

**Comment by havenwood** [comment]
Assuming you're using Homebrew, first make sure to install Jemalloc.
```sh
brew install jemalloc
```
Then add `-- --with-jemalloc` following your `ruby-install` command, like shown in the `ruby-3.2` example below.
```sh
ruby-install --update ruby-3.2 -- --with-jemalloc
```

**Comment by Z2Flow** [comment]
Thanks Shannon
